lib/files.dart

```diff
@@ -0,0 +1,18 @@
+bool isLocalFilePath(String path) => !path.contains('http');
```

Comment:
i think this is not good enough.

maybe user wants to download image from url 'www.xyz.bla/123.jpg'
or Camera returns temp image which has http in its temp name.

Reply:
I covered second case that could be an issue.